src/cs/Ssh/SshSession.cs

```diff
@@ -494,6 +501,35 @@ await SendMessageAsync(KeyExchangeInitMessage.None, connectCts.Token)
 		await this.connectCompletionSource.Task.ConfigureAwait(false);
 	}
 
+	internal void ImplementKeepAlive(CancellationToken cancellation)
+	{
+		if (Config.KeepAliveTimeoutInSeconds > 0)
+		{
+			keepAliveTimer = new (
+				async (state) =>
+				{
+					if (this.CanAcceptRequests)
```

Comment:
Should we do try-catch? if this lambda is `async void`, an exception here may terminate the process.